tests/system/test_marathon_root.py

```diff
@@ -60,6 +60,7 @@ def marathon_service_name():
 
 
 def setup_module(module):
+    common.wait_for_marathon_about()
     common.cluster_info()
```

Comment:
add the `cluster_info` already calls the marathon about.
https://github.com/mesosphere/marathon/blob/master/tests/system/common.py#L109-L114

It think if we just call a wait for marathon function at setup prior to cluster_info we are good.   We used to have just a wait for marathon... perhaps it was refactored out.

Reply:
well the cluster info prints out marathon version as a part of cluster info so I think it is still useful...

I refactored the wait to the setup phase because it was suggested by @zen-dog and I actually liked that suggestion. So now we call it twice but I guess it does not matter